File: Module/Home/src/main/java/com/module/home/game/model/RecommendRoomModel.java
package com.module.home.game.model;

import com.component.busilib.friends.RecommendModel;

import java.util.List;

public class RecommendRoomModel {

    List<RecommendModel> mRoomModels;

    public RecommendRoomModel(List<RecommendModel> list) {
        this.mRoomModels = list;
    }

    public List<RecommendModel> getRoomModels() {
        return mRoomModels;
    }

    public void setRoomModels(List<RecommendModel> roomModels) {
        mRoomModels = roomModels;
    }

}

File: Module/PlayWays/src/main/java/com/module/playways/friendroom/RecommendRoomModel.kt
package com.module.playways.friendroom

import com.alibaba.fastjson.annotation.JSONField
import com.component.busilib.model.PartyRoomInfoModel
import com.module.playways.mic.home.RecommendMicInfoModel
import com.module.playways.mic.home.RecommendMicRoomModel
import java.io.Serializable

class RecommendRoomModel : Serializable {
    companion object {
        const val EGST_STAND = 1  //抢唱房间
        const val EGST_MIC = 2    //排麦房间
        const val EGST_PARTY = 3  //主题房间
    }

    @JSONField(name = "gameSceneType")
    var gameSceneType: Int = 0
    @JSONField(name = "micRoomList")
    var micRoom: RecommendMicInfoModel? = null
    @JSONField(name = "standRoomList")
    var grabRoom: RecommendGrabRoomModel? = null
    @JSONField(name = "partyRoomList")
    var partyRoom: PartyRoomInfoModel? = null

    override fun toString(): String {
        return "RecommendRoomModel(gameSceneType=$gameSceneType, micRoom=$micRoom, grabRoom=$grabRoom)"
    }

}